heartbeat/tracer/tracer.go

```diff
@@ -63,28 +70,46 @@ func NewSockTracer(path string, wait time.Duration) (st SockTracer, err error) {
 	return st, nil
 }
 
-func (st SockTracer) Write(message string) error {
+func (st SockTracer) Start() {
+	err := st.write(MSG_START)
+	logp.L().Errorf("could not write start trace message: %s", err)
```

Comment:
dont we need err != nil check?  Would it not log when its nil?

Reply:
🤦 yes